### ISSUE TITLE
replace phylogeny qza files

### ISSUE THREAD

**Issue by gregcaporaso** [qiime2/q2-types#31]
The current ``tests/data/phylogeny-rooted.qza`` and ``tests/data/phylogeny-unrooted.qza`` files are really big and don't work with the wiki tutorial. We should replace them with these files:

https://dl.dropboxusercontent.com/u/2868868/phylogeny-rooted.qza
https://dl.dropboxusercontent.com/u/2868868/phylogeny-unrooted.qza

It would be nice to re-write the history to remove the current files that are in there, since they are much larger than anything else. 

Thanks @jairideout for catching this issue!

**Comment by jairideout** [comment]
👍 , I'll work on rewriting the history and adding these files.

**Comment by jairideout** [comment]
The incompatible/large phylogeny .qza files were removed in #32. I rewrote the git history to remove those blobs (pushed to upstream, no PR) using [BFG Repo Cleaner](https://rtyley.github.io/bfg-repo-cleaner/). The compatible/small phylogeny .qza files were added in #33.

The old repo size was 18.2 MB. The new size is 5.6 MB.

Existing forks of this repo will need to be deleted and re-forked to avoid reintroducing the old git history. A rebase on your fork should be possible, but the easiest way is to delete and refork.

Developers merging PRs against this repo should make sure the old history isn't accidentally reintroduced to the repo.

I will contact developers on the QIIME 2 Slack #developers channel asking everyone to refork.
